test/test_gipc.py

```diff
@@ -1365,7 +1371,7 @@ def test_orphaned_signal_watcher(self):
                 assert p.exitcode == -signal.SIGTERM
             else:
                 assert p.exitcode == signal.SIGTERM
-        s.cancel()
+        gevent.signal.signal(signal.SIGTERM, s)
```

Comment:
seems like this change broke Windows CI : ) maybe just revert?

Reply:
yeah, I reverted and it seems like it still passes on osx. It was failing previously though, not sure why but I guess it ended up being unnecessary.